src/lib/schedule/Schedule.js

```diff
@@ -162,11 +162,16 @@ class Schedule {
 	 * @returns {this}
 	 */
 	async delete(id) {
-		const _task = this._tasks.find(entry => entry.id === id);
-		if (!_task) throw new Error('This task does not exist.');
+		const task = this._tasks.find(entry => entry.id === id);
+		if (!task) {
+			const possibleIndex = this.tasks.findIndex(entry => entry.id === id);
```

Comment:
duplicate logic, you need possibleIndex and splice regardless of `!task` or `task`

Reply:
Fixed